modules/install-roxctl-cli-linux.adoc

```diff
@@ -7,21 +7,27 @@
 
 You can install the `roxctl` CLI binary on Linux by using the following procedure.
 
+[NOTE]
+====
+`roxctl` CLI for Linux is available for `x86_64`, `amd64`, `ppc64le`, and `s390x` architectures.
```

Comment:
`x86_64` is the same as `amd64`. These are two different ways to name the same thing. Unfortunately, one is used in certain places (e.g. `uname -m` returns `x86_64`) whereas the other in different ones (e.g. CLOUDDST team convention is to use `amd64`).

Following the Red Hat convention to prefer `amd64` over `x86_64`, this should say:

```suggestion
`roxctl` CLI for Linux is available for `amd64`, `ppc64le`, and `s390x` architectures.
```